### DISCUSSION TITLE
Forwards or backwards synchronization cannot locate the row.

### DISCUSSION THREAD

**Discussion by note286** [lvjr/tabularray#325] [Q&A]
```latex
\documentclass{article}
\usepackage{tabularray}
\begin{document}
\begin{tblr}{colspec={cc}}
a & a \\
a & a \\
a & a \\
\end{tblr}

\begin{tabular}{cc}
b & b \\
b & b \\
b & b \\
\end{tabular}
\end{document}
```

I found that in `tblr` environment, no matter forward synchronization or reverse synchronization, it can't locate rows, only whole tables. While the `tabular` environment can be positioned to the row. Is this a bug or a temporarily unavailable function?

**Comment by lvjr** [comment]
It is the same for `tabularx` and `align` environments. See https://tex.stackexchange.com/questions/648337/synctex-dont-work-with-tabularray-package

**Reply by note286** [reply]
Thank you very much!
